src/tests/src/solver/modelParser/testModelParser.cpp

```diff
@@ -312,12 +312,86 @@ BOOST_AUTO_TEST_CASE(test_library_model_multiple_variables)
     Antares::Solver::ModelParser::Library libraryObj = parser.parse(library);
     BOOST_REQUIRE_EQUAL(libraryObj.models.size(), 1);
     BOOST_REQUIRE_EQUAL(libraryObj.models[0].variables.size(), 2);
-    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[0].name, "var_name1");
-    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[0].lower_bound, 0);
-    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[0].upper_bound, 1);
-    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[1].name, "var_name2");
-    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[1].lower_bound, -1);
-    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[1].upper_bound, 2);
+    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[0].id, "var_name1");
+    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[0].lower_bound, "0");
+    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[0].upper_bound, "1");
+    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[1].id, "var_name2");
+    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[1].lower_bound, "-1");
+    BOOST_CHECK_EQUAL(libraryObj.models[0].variables[1].upper_bound, "2");
+}
+
+// variable bounds are strings expressions
+BOOST_AUTO_TEST_CASE(test_library_model_variables_bounds)
```

Comment:
_Really cosmetic_ : 
in order to highlight the unit test pattern **arrange, act, assert**, we could separate the action 
```c++
Antares::Solver::ModelParser::Library libraryObj = parser.parse(library);
```
from the two other steps with empty lines.